src/config/keys.ts

```diff
@@ -42,6 +42,12 @@ export const buildS3FileContentKey = (id?: UUID) => [ITEMS_KEY, id, 'content'];
 export const ITEM_FLAGS_KEY = 'itemFlags';
 export const buildItemFlagsKey = (id: UUID) => [ITEMS_KEY, id, 'flags'];
 
+export const CATEGORY_TYPES_KEY = 'categoryTypes'
+export const CATEGORY_INFO = 'categoryInfo';
+export const buildCategoriesKey = (id?: UUID[]) => ['Categories', id?.toString()]
+export const buildItemCategoryKey = (id?: UUID) => [ITEMS_KEY, id, 'Categories'];
```

Comment:
I guess `buildCategoriesKey` is using type ids ? Maybe change for a clearer name, and use `hashItemsIds` to change your array into a string. It will reorder the ids as well and hit the same key even if you provide type ids in a different order!